Readme.md

```diff
@@ -72,7 +72,6 @@ These instructions have been tested on Debian Jessie.
     docker-compose logs -f snoop-worker
 
     docker-compose run --rm search ./manage.py addcollection testdata http://snoop/collections/testdata/json --public
-    docker-compose run --rm search ./manage.py resetindex testdata
```

Comment:
`addcollection` should not clobber your elasticsearch index without warning. Either be explicit about it, or somehow auto-detect that the index is missing and create it.

Reply:
It won't clobber it, i'll nuke it.